### PR TITLE
[YUNIKORN-2628] revert relative path for release announcement

### PR DESCRIPTION
**Pull request by wilfred-s**
### What is this PR for?
Fix broken links: remove `..`

### What type of PR is it?
* [X] - Bug Fix

### What is the Jira issue?
* https://issues.apache.org/jira/browse/YUNIKORN-2628

### How should this be tested?
Link on download page to the release announcement should work and not show 404

### CODE REVIEW

**Review by pbacsko — APPROVED** [review]
+1

File: src/pages/community/download.md [hunk 1]
```diff
@@ -33,11 +33,11 @@ We publish prebuilt docker images for everyone's convenience.
...
+| v1.5.0  | 2024-03-14   | [Download](https://www.apache.org/dyn/closer.lua/yunikorn/1.5.0/apache-yunikorn-1.5.0-src.tar.gz)<br/>[Checksum](https://downloads.apache.org/yunikorn/1.5.0/apache-yunikorn-1.5.0-src.tar.gz.sha512) & [Signature](https://downloads.apache.org/yunikorn/1.5.0/apache-yunikorn-1.5.0-src.tar.gz.asc) | [scheduler](https://hub.docker.com/layers/apache/yunikorn/scheduler-1.5.0/images/sha256-9cefd0df164b9c4d39f9e10b010eaf7d8f89b130de1648e94f75b9b95d300a00)<br/>[admission-controller](https://hub.docker.com/layers/apache/yunikorn/admission-1.5.0/images/sha256-28f63ec17ac96faa08d79a5e133a2728bc339319f4b71d481f3c0e6477944697)<br/>[web](https://hub.docker.com/layers/apache/yunikorn/web-1.5.0/images/sha256-ccdc9d1e4cbc36037e181495f48e569bfc43a24f65efbd52d8ddc7183f1f3e17)<br/>[scheduler plugin](https://hub.docker.com/layers/apache/yunikorn/scheduler-plugin-1.5.0/images/sha256-01f1a6a8aab81aa3b385e4d439a1b919174626475bd8939af8c944f962170d05) | [Announcement](/release-announce/1.5.0) |
```
[the remark this answers is not in the document]
Comment (wilfred-s) [reply src/pages/community/download.md:38]: We have reverted to what it was before [YUNIKORN-2595](https://issues.apache.org/jira/browse/YUNIKORN-2595) was committed. I have tested the lates change on Chrome, Safari, Opera and Firefox and all show working links. Please make sure that you have done a hard reload as often the pages get cached
FF-link-shot

[the remark this answers is not in the document]
Comment (chia7712) [reply src/pages/community/download.md:38]: Sorry that I use incorrect way to test it. Please ignore my previous comment.

And now I use `./local-build.sh run` to test it again. The master branch works well on my firefox. 

螢幕快照 2024-05-16 18-26-36

How you get the incorrect link? please share it to me, thanks

[the remark this answers is not in the document]
Comment (wilfred-s) [reply src/pages/community/download.md:38]: I have noticed that caching often gets in the way.
When I test these changes I start  the dev site with `./local-build.sh run` make it all come up and ready for browsing. Open the page I want to change in an editor and in the browser.
Make the change(s) and save the file. Wait for the rebuild to finish and then look if the page shows the right detail. Sometimes I will trigger a reload by pressing the reload icon holding the shift key (on MAC) to force a cache bypass.

[the remark this answers is not in the document]
Comment (chia7712) [reply src/pages/community/download.md:38]: Thanks for the sharing!

**Pull request closed without merging**